spylon_kernel/init_spark_magic.py

```diff
@@ -37,8 +39,9 @@ def cell_init_spark(self):
 
         globals_dict = self.env
         exec(self.code, globals_dict)
+        application_name = globals_dict.get('application_name', None)
```

Comment:
How about a global `DEFAULT_APPLICATION_NAME` in `scala_interpreter.py` that can be used in both files? I just used `"ScalaMetaKernel"` as it's the `init_spark_session` default.